addons/binding/org.openhab.binding.yamahareceiver.test/README.md

```diff
@@ -0,0 +1,2 @@
+# Tests for the Yamaha Binding
```

Comment:
There should be a new line, not that this ends up in the readme but for completion it should be great.